src/Service/Users.php

```diff
@@ -49,7 +49,8 @@ class Users {
   public function __construct(ConfigFactory $configFactory,
                               OpenamApiClient $openamApiClient,
                               EventDispatcherInterface $eventDispatcher) {
-    $this->config = $configFactory->get('openam_api_operations.settings');
+    $openamConfig = $configFactory->get('openam_api.settings');
```

Comment:
This change needs to be also made on https://github.com/dakkusingh/openam_api/blob/b0d34c7f80afeafd6160e64783d418195e659e1b/src/Service/Users.php#L37

Reply:
Updated.